Makefile

```diff
@@ -1,5 +1,11 @@
 .PHONY: all deps test validate lint
 
+# Which github repostiory and branch to use for testing with skopeo
+SKOPEO_REPO = projectatomic/skopeo
+SKOPEO_BRANCH = master
+# Set SUDO=sudo to run container integration tests using sudo.
+SUDO =
```

Comment:
I am ambivalent about the default value. Personally `SUDO = sudo` would make my life easier, but the empty string default is less surprising, and on systems which do not need root privileges to use Docker this does not gain unnecessary privileges and is not unexpectedly interactive.